File: app/src/main/java/com/ajdi/yassin/popularmovies/data/local/model/CreditsResponse.java
package com.ajdi.yassin.popularmovies.data.local.model;

import com.google.gson.annotations.SerializedName;

import java.util.List;

/**
 * @author Yassin Ajdi
 * @since 11/12/2018.
 */
public class CreditsResponse {

    @SerializedName("cast")
    private List<Cast> cast;

    public List<Cast> getCast() {
        return cast;
    }

    public void setCast(List<Cast> cast) {
        this.cast = cast;
    }
}

File: app/src/main/java/com/ajdi/yassin/popularmovies/data/local/model/CreditsResponse.kt
package com.ajdi.yassin.popularmovies.data.local.model

import com.google.gson.annotations.SerializedName

/**
 * @author Yassin Ajdi
 * @since 11/12/2018.
 */
data class CreditsResponse (

    @SerializedName("cast")
    var cast: List<Cast>? = null
)
